Entity/SelectedStopPointRepository.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace CanalTP\MttBundle\Entity;
+
+use Doctrine\ORM\EntityRepository;
+
+/**
+ * SelectedStopPointRepository
+ *
+ * This class was generated by the Doctrine ORM. Add your own custom
+ * repository methods below.
+ */
+class SelectedStopPointRepository extends EntityRepository
```

Comment:
Yes but because I saw that all repositories are created even if they are empty, I did the same :) So I will remove all empty repositories I created.